docs/content/docs/changelogs/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: "Changelogs"
+title: "What's new"
 draft: false
 area: changelogs
```

Comment:
@heliocliu What do you think about renaming the area (and the URL) to "updates?"

Reply:
that sounds good to me.  definitely more human-friendly

Reply:
Changed.